spec/jara/releaser_spec.rb

```diff
@@ -171,6 +171,14 @@ module Jara
         components[3].should == master_sha[0, 8]
       end
 
+      it 'overrides app name with the :app_name option' do
```

Comment:
Ah, that's much better. I initially placed the test in the context for `#release` and attempted to follow the naming that was used there.